src/Compiler/Driver/CompilerConfig.fs

```diff
@@ -30,23 +30,22 @@ open FSharp.Compiler.BuildGraph
 open FSharp.Core.CompilerServices
 #endif
 
-let (++) x s = x @ [s]
+let (++) x s = x @ [ s ]
 
 //----------------------------------------------------------------------------
 // Some Globals
 //--------------------------------------------------------------------------
 
-let FSharpSigFileSuffixes = [".mli";".fsi"]
+let FSharpSigFileSuffixes = [ ".mli"; ".fsi" ]
 
-let mlCompatSuffixes = [".mli";".ml"]
+let FSharpMLCompatFileSuffixes = [ ".mli"; ".ml" ]
 
-let FSharpImplFileSuffixes = [".ml";".fs";".fsscript";".fsx"]
+let FSharpImplFileSuffixes = [ ".ml"; ".fs"; ".fsscript"; ".fsx" ]
 
-let FSharpScriptFileSuffixes = [".fsscript";".fsx"]
+let FSharpScriptFileSuffixes = [ ".fsscript"; ".fsx" ]
 
-let doNotRequireNamespaceOrModuleSuffixes = [".mli";".ml"] @ FSharpScriptFileSuffixes
```

Comment:
I presume, that was on purpose, as part of having those in the `FSharpMLCompatFileSuffixes`?

Reply:
I effectively moved this logic to here: https://github.com/dotnet/fsharp/pull/13195/files#diff-efdad7fa53b23cfadc0c316627cd7f8833d0a531db500cc7c220cc4fa0465ab2R124